cloudhub-1/modules/ROOT/Edits/vpn-maintenance.adoc

```diff
@@ -0,0 +1,71 @@
+= Anypoint VPN Maintenance
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:page-aliases: runtime-manager::vpc-maintenance.adoc,
+
+
+== Anypoint VPN Update
+
+MuleSoft Anypoint VPN is a managed service that is underpinned by AWS Site-to-Site VPN connections. VPN is key for any integrations or flows that require connectivity to your on-premises or other external platforms. Updating an Anypoint VPN that is not set up with at least a pair of tunnels using BGP routing will cause downtime. See xref:vpn-high-availability.adoc[Anypoint VPN High Availability] for more information.
+
+After the October 2022 quarterly maintenance window, Anypoint VPN will no longer support quarterly customer self-upgrades. From this point forward, updates made to Anypoint VPN will adhere to AWS's standard continuous upgrade process. Upgrades will be applied on-demand, with no published maintenance window schedule, no customer opt-in self-upgrade capabilities, with *potential (limited) downtime for customers that do not have a* xref:vpn-high-availability.adoc[High Availability (HA) VPN implementation].
+
+Deploy a xref:vpn-high-availability.adoc[high-availability VPN] solution to avoid any downtime associated with these AWS-delivered security updates. Review the below section for recommendations and troubleshooting.
```

Comment:
```suggestion
Deploy a xref:vpn-high-availability.adoc[high-availability VPN] solution to avoid any downtime associated with these AWS-delivered security updates. Review the following sections for recommendations and troubleshooting information.
```